app/src/main/java/woowacourse/movie/ui/alarm/NotificationCreator.kt

```diff
@@ -0,0 +1,72 @@
+package woowacourse.movie.ui.alarm
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import androidx.core.app.NotificationCompat
+import woowacourse.movie.R
+import woowacourse.movie.model.MovieTicketModel
+import woowacourse.movie.ui.seat.SeatSelectionActivity
+import woowacourse.movie.ui.ticket.MovieTicketActivity
+
+class NotificationCreator(context: Context) {
+    private val notificationManager: NotificationManager by lazy {
+        context.getSystemService(
+            Context.NOTIFICATION_SERVICE,
+        ) as NotificationManager
+    }
+
+    fun createNotificationChannel() {
+        val notificationChannel = NotificationChannel(
+            CHANNEL_ID,
+            ALARM_CHANNEL_NAME,
+            NotificationManager.IMPORTANCE_HIGH,
+        )
+        notificationManager.createNotificationChannel(
+            notificationChannel,
+        )
+    }
+
+    fun makeNotification(context: Context, movie: MovieTicketModel) {
```

Comment:
```suggestion
    fun makeNotification(movie: MovieTicketModel) {
```
context를 파라미터로 전달받기보다는, 생성자로 캡슐화한 context를 활용해보면 어떨까요?